src/Data/PropertyHolder.php

```diff
@@ -72,6 +72,7 @@ public function getPropertyType($propertyName)
             'RenderingEngine_Maker',
             'Parent',
             'PropertyName',
+            'CDF',
```

Comment:
@asgrim `CDF` is a property which was part of the browscap.ini some time ago, I asume before you took over the project lead. I dont known what whis property is about.
If you are installing an Xampp like I did an outdated browscap.ini was part of that installation.
Adding this property here should make sure, that the unittests for this library do not fail with an outdated ni file.